packages/components-react/projects/react-wrapper/src/hooks.ts

```diff
@@ -30,6 +30,14 @@ export const usePrefix = (tagName: string): string => {
   }
 };
 
+export const useTheme = (): Theme => {
+  if (process.env.NODE_ENV === 'test' && skipCheck) {
```

Comment:
why is this necessary?

Reply:
Because of the opt-out via `skipCheckForPorscheDesignSystemProviderDuringTests()`.  When this is called, there is no context and you will run into an exception.